src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -105,6 +109,10 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 			setInternalOptions();
 		}
 
+		if (fieldDef.inputSettings?.selectLimit === 0) {
+			setCheckboxListDisabled(true)

```

Comment:
Please don't forget the ";" at the end of each line. It might not be as important in javascript but in typescript it can lead to production errors.